README.md

```diff
@@ -190,6 +194,7 @@ Give a ⭐️ if you like this project!
 - Thanks to the Microverse team for the great curriculum.
 - Thanks to the Code Reviewer(s) for the insightful feedback.
 - Thanks to our coding partners, the morning session team and standup team for their coperation.
+- I would like to ackowledge Nelson Sakwa the author of the original design,
```

Comment:
- [ ] 👍 Great job acknowledging the author of the design! To provide users with easy access to the actual design, consider adding a link to the design source. This way, users can quickly reference the design you're mentioning and gain a better understanding of the context. Keep up the good work in enhancing the user experience! 👏🔗🎨

Reply:
- [ ] Again as requested by the previous reviewer, consider adding a link to the design source to provide users with easy access to the actual design.